Tests/run_tests.py

```diff
@@ -173,7 +170,6 @@ def is_numpy():
 # Skip Bio.bgzf doctest for broken gzip, see http://bugs.python.org/issue17666
 def _have_bug17666():
     """Debug function to check if Python's gzip is broken (PRIVATE).
-
```

Comment:
Please restore this blank line which is a deliberate part of the docstring for private function ``_have_bug17666`` - see https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings